lib/models/manifest.rb

```diff
@@ -1,3 +1,56 @@
+require_relative '../query_dsl'
+
 class Manifest < ActiveRecord::Base
+  include Elasticsearch::Model
+
+  ActiveRecord::Base.raise_in_transactional_callbacks = true
+
+  after_commit on: [:create] do
```

Comment:
thoughts on doing this reindexing at `lib/app.rb` instead of in callbacks? always nice to avoid callbacks...